pkg/api/topic_api_handers.go

```diff
@@ -0,0 +1,67 @@
+package api
+
```

Comment:
@restevens402 this creates a new topic on the fly and takes a `topicName` and initializes a new `TopicHandler`

This forms the start of out chat topics